packages/@aws-cdk/core/lib/expires.ts

```diff
@@ -0,0 +1,47 @@
+import { Duration } from './duration';
+/**
+ * Represents a date of expiration.
+ *
+ * The amount can be specified either as a Date object, timestamp, Duration or string.
+ */
+export class Expires {
```

Comment:
I think this should be called `Expiration` rather than `Expires`. Class names should be nouns.